src/less/dropdowns.less

```diff
@@ -452,13 +452,9 @@
   text-align: center;
 
   .dropdown {
-    .date-range__buttons {
-      .button {
-        background-color: @global-color-gray-lightest;
-        box-shadow: none;
-        font-size: @global-font-size-small;
-        height: @global-whitespace-large;
-      }
+    .button {
```

Comment:
Won't this affect nested button classes for all dropdowns on the app?

Reply:
I've seen the code.. this style is already given to all dropdown except having class date-range__buttons(i didn't find this class anywhere so missed by someone i guess) and in analytics page, and appointment manager same style is given. That's why I made it globally.
![Uploading Screen Shot 2019-08-16 at 9.39.13 AM.png…]()
![Screen Shot 2019-08-16 at 9 45 27 AM](https://user-images.githubusercontent.com/10115528/63143158-9e53aa80-c00a-11e9-854d-351f13f6dbfa.png)
![Screen Shot 2019-08-16 at 9 46 04 AM](https://user-images.githubusercontent.com/10115528/63143178-a7dd1280-c00a-11e9-9ab4-f880e36b2072.png)